recipes/google-cloud-cpp-compute/bld.bat

```diff
@@ -0,0 +1,25 @@
+@echo on
+setlocal EnableDelayedExpansion
+
+:: CMake does not like paths with \ characters
+set LIBRARY_PREFIX="%LIBRARY_PREFIX:\=/%"
+set BUILD_PREFIX="%BUILD_PREFIX:\=/%"
+set SRC_DIR="%SRC_DIR:\=/%"
+
+cmake -G "Ninja" ^
+    -S . -B .b ^
```

Comment:
Would prefer to avoid names with leading dots, especially on windows.

Reply:
Are you concerned about folks building the code on FAT filesystems with only 8.3 filename support?  I am not sure anything would work in that case, but no matter, changed.